RevenueCat/Plugins/Editor/RevenueCatDependencies.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <dependencies>
     <androidPackages>
-        <androidPackage spec="com.revenuecat.purchases:purchases-hybrid-common:1.6.1" />
+        <androidPackage spec="com.revenuecat.purchases:purchases-hybrid-common:[1.6.1]" />
```

Comment:
🤯 I didn't expect this to be needed based on the documentation here https://github.com/googlesamples/unity-jar-resolver#android-resolver-usage. 
did you find docs for this?

Reply:
Me neither. But take a look at that spectrum conversation, someone found it in another plugin's dependencies

Reply:
that's nuts. we should open up an issue on `https://github.com/googlesamples/unity-jar-resolver` asking them to document that behavior

Reply:
yes, good point I am opening an issue